templates/_layouts/_footer.html

```diff
@@ -11,6 +11,9 @@
         <li class="p-list__item">
           <a href="https://github.com/canonical/canonical-design/issues/new">Report a bug</a>
         </li>
+        <li class="p-list__item">
+          <a class="js-revoke-cookie-manager" href="">Manage your tracker settings</a>
```

Comment:
```suggestion
          <a class="js-revoke-cookie-manager" href="#">Manage your tracker settings</a>
```
I would hash this. If a user doesn't have JS enabled they will be redirected to the homepage which might be misleading.
We actually need to add a no-JS fallback for the cookie policy as currently it relies on JS. I have made a maintenance issue for this